neuralmonkey/readers/plain_text_reader.py

```diff
@@ -6,21 +6,30 @@
 from neuralmonkey.logging import warn
 
 
-def get_plain_text_reader(encoding: str = "utf-8") -> Callable[[List[str]],
-                                                               Iterable
-                                                               [List[str]]]:
-    """Get reader for space-separated tokenized text."""
-    def reader(files: List[str]) -> Iterable[List[str]]:
+def lines_reader(encoding: str = "utf-8") -> Callable[[List[str]],
+                                                      Iterable[str]]:
+    def reader(files: List[str]) -> Iterable[str]:
         for path in files:
-
             if path.endswith(".gz"):
                 with gzip.open(path, 'r') as f_data:
                     for line in f_data:
-                        yield str(line, 'utf-8').strip().split(" ")
+                        yield str(line, 'utf-8')
             else:
                 with open(path, encoding=encoding) as f_data:
                     for line in f_data:
-                        yield line.strip().split(" ")
+                        yield line
+
+    return reader
+
+
+def get_plain_text_reader(encoding: str = "utf-8") -> Callable[[List[str]],
```

Comment:
"Komplikace" to neni. Tady by to autora kódu spíš mělo přimět k tomu, aby si uvědomil, že ``Callable[[List[str]], Iterable[List[str]]]`` se dá nazvat ``FileReader``, pak to nahoře v tomhle modulu takhle zadefinoval a udělal volání týhle funkce takhle:

```python
def get_plain_text_reader(encoding: str = "utf-8") -> FileReader:
    # (...)
```